### DISCUSSION TITLE
[Feature]: Does pim officially support configuring the environment in wsl2 and then accessing it through Win10?

### DISCUSSION THREAD

**Discussion by stuffren** [pimcore/pimcore#16760] [Q&A / Support]
### Feature description

I'm trying to access pim from a windows 10 browser on ubuntu 22.04 WSL by creating a symbolic link.
To achieve this I did the following:
1. create a new pim directory in window10 and then create a symbolic link on wsl2
sudo ln -s /mnt/d/sites/pim/public /var/www/pim
2. Create a new configuration file
sudo touch /etc/apache2/sites-available/pim.local.conf
Configure the pim.local.conf file as:
<VirtualHost *:80
        ServerName www.pim.local
        ServerAlias pim.local
        ServerAdmin webmaster@localhost
        DocumentRoot /var/www/pim

        <Directory /var/www/>
               Options Indexes FollowSymLinks MultiViews
               AllowOverride all
               Order allow,deny
               Allow from all
        </Directory

        ErrorLog /var/log/apache2/error.pim.local.log
        CustomLog /var/log/apache2/error.pim.local.log combined
</VirtualHost
3. Activate the new configuration
4. Restart the Apache service
5. Set the virtual domain name in the C:\Windows\System32\drivers\etc directory and open the hosts file and add:
::1 www.pim.local
127.0.0.1 www.pim.local
::1 pim.local
127.0.0.1 pim.local
Can access pim.local, but end up getting a 500 error from pim's server on the home page. The prompt is as follows:
An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.
![01](https://github.com/pimcore/pimcore/assets/136626456/d1512a3a-fdc1-4a3d-b2df-29592d62852b)

Does pim officially support configuring the environment in wsl2 and then accessing it through Win10?

**Comment by SamyMP** [comment]
Hello,

Did you follow the pimcore installation procedure ? Especially this step : https://pimcore.com/docs/platform/Pimcore/Getting_Started/Installation/Webserver_Installation#4-launch-installer

**Comment by khenow1978** [comment]
Why would you do something like that? Is there any benefit from just installing Pimcore into the WSL?

That's what I did. I run Windows 11 and WSL with Ubuntu 22.04. I just installed Pimcore within WSL and followed the installation instructions. Windows Docker Desktop ist connected with WSL and sees all the containers created by Pimcore.

So basically what I do is: I start the Docker Containers in Docker Desktop, open edge and type "localhost/admin". That's pretty straightforward.. why would you want to do it differently?

**Comment by stuffren** [comment]
> Why would you do something like that? Is there any benefit from just installing Pimcore into the WSL?
> 
> That's what I did. I run Windows 11 and WSL with Ubuntu 22.04. I just installed Pimcore within WSL and followed the installation instructions. Windows Docker Desktop ist connected with WSL and sees all the containers created by Pimcore.
> 
> So basically what I do is: I start the Docker Containers in Docker Desktop, open edge and type "localhost/admin". That's pretty straightforward.. why would you want to do it differently?

Thank you for your answer, mainly because I have not used Docker yet. Ha-ha. Looks like I'm going to have to try the same way you installed it with docker.
I usually develop PHP applications locally, and generally use wampserver. Many mainstream out-of-the-box PHP applications (such as wordpress, opencart, grav...). Are supported in Windows under the wamp installation and development, this you know. However, pim is only supported on linux, and it is not a PHP application out of the box, so it was quite a challenge for me to install pim in liunx.

**Comment by stuffren** [comment]
> Hello,
> 
> Did you follow the pimcore installation procedure ? Especially this step : https://pimcore.com/docs/platform/Pimcore/Getting_Started/Installation/Webserver_Installation#4-launch-installer

Thanks for your reply. According to the URL you provided, I found that I may have made a mistake in two programs. I install the pim, its installation document address is: https://docs.akeneo.com/master/install_pim/manual/index.html
However, the address you provided is the pimcore installation tutorial address.
What I want to confirm is whether akeneo and pimcore are open source products of the same organization?
I think I should ask questions in akeneo's feedback section. I'm really sorry!
